api/SVGElement.json

```diff
@@ -201,40 +201,40 @@
           "spec_url": "https://svgwg.org/svg2-draft/interact.html#ErrorEvent",
```

Comment:
This should be possible to test with a [structurally external element](https://svgwg.org/svg2-draft/struct.html#TermStructurallyExternalElement), and `<image>` is probably the best since it's always been supported:
```svg
<svg width="200" height="200" xmlns="http://www.w3.org/2000/svg">
  <script>
  // <![CDATA[
  document.addEventListener('error', function(event) {
    alert(event);
  }, true);
  // ]]>
  </script>
  <image href="404.png" height="200" width="200"/>
</svg>
```

Can you confirm the data with a test like that?